spec/models/post_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe Post, :type => :model do
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)